docs/cli/Troubleshooting.md

```diff
@@ -25,5 +25,5 @@ You can uninstall previous versions of the cli with:
 ```bash
 npm uninstall --global ignite-cli
 # or
-yarn global remove ignite-cli
+pnpm remove ignite-cli -g
```

Comment:
I might add this as another `# or` line rather than remove the `yarn` one. I think npm globals are probably the most common anyway